core/nodejs18Action/Dockerfile

```diff
@@ -36,6 +36,7 @@ RUN apt-get update \
     ca-certificates \
     # For the proxy.
     unzip \
+    python \
```

Comment:
should this be `python` or `python3`?

Reply:
I don't know if it matters tbh 😅 . The script itself should be compatible with both and we know that `python` works in Node:14, so I'm inclined to keep it that way.